server/models/referralProbationPractitionerEmail.ts

```diff
@@ -0,0 +1,3 @@
+export interface AmendProbationPractitionerEmailUpdate {
+  ppEmail: string
```

Comment:
I noticed in the service, the dto included:
val preventEmailNotification: Boolean? = null

Should this be included here, or not in the service?
I may have missed something though.

Reply:
It was required by the interface, but I don't have any guidance either way if it's needed or not.  I've assumed we include it but default to null in this case and add an actual value if it becomes necessary in a later change.

@rgforsyth

Reply:
👍